<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Egresados</class>
 <widget class="QWidget" name="Egresados">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>621</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1111</width>
    <height>621</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f5f6fa;
color: #2d2d2d;</string>
  </property>
  <widget class="QToolButton" name="btnExportar_egresados">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
   background-color: #2980b9;
    color: #FFFFFF;
    border: none;
    padding: 8px 7px;
    border-radius: 15px;
}
QToolButton:hover {
	background-color: #0D47A1;
}

/* --- Estilo del menú desplegable --- */
QMenu {
    background-color: white;         /* fondo blanco */
    color: black;                    /* texto negro */
    border: 1px solid #c0c0c0;
}

QMenu::item {
    padding: 5px 20px;
}

QMenu::item:selected {
    background-color: #0078d7;       /* azul Windows */
    color: white;                    /* texto blanco al seleccionar */
}</string>
   </property>
   <property name="text">
    <string>Exportar</string>
   </property>
   <property name="icon">
    <iconset resource="../icons/resources_ui.qrc">
     <normaloff>:/icons/exportar.png</normaloff>:/icons/exportar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="popupMode">
    <enum>QToolButton::ToolButtonPopupMode::InstantPopup</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frameTabla_egresados">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>1061</width>
     <height>481</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>900</width>
     <height>450</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>500</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frameTabla_student {
    border: 1px solid #d5dbdb;
    border-radius: 12px;
    background-color: white;
	border-radius: 10px
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableView" name="tableW_egresados">
      <property name="styleSheet">
       <string notr="true">QTableView {
    background-color: #F7F9FC;
    gridline-color: #E3E8EF;
    color: #0B1321;
    alternate-background-color: #E3F2FD;
    selection-background-color: #2980b9;
    selection-color: #FFFFFF;
    border: 1px solid #CBD5E1;
}

/* Cabecera horizontal */
QHeaderView::section {
    background-color: #ECF0F1;   /* encabezado */
    color: #2C3E50;              /* azul oscuro */
    font-weight: bold;
    border: none;
    padding: 6px;
}

/* Cabecera vertical */
QHeaderView::section:vertical {
    background-color: #ECF0F1;
    color: #2C3E50;
    font-weight: bold;
    border: none;
    padding: 4px;
}

/* Botón de esquina (arriba a la izquierda) */
QTableCornerButton::section {
    background-color: transparent;
    border: none;
}

/* Scrollbar vertical */
QScrollBar:vertical {
    background: transparent;
    width: 10px;
    margin: 2px;
}

QScrollBar::handle:vertical {
    background: #CBD5E1;
    border-radius: 4px;
    min-height: 20px;
}

QScrollBar::handle:vertical:hover {
    background: #94A3B8;
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    background: none;
    border: none;
    height: 0px;
}</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lblTitulo_logo_estu">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>0</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>19</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #2d2d2d</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="text">
    <string>Módulo de egresados</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="indent">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnActualizar_db_egresados">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>30</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Questrial</family>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   background-color: #7f8c8d;
    color: #F3F6FF;
    border: none;
    padding: 8px 16px;
    border-radius: 15px;
	font-family: &quot;Questrial&quot;;
}
QPushButton:hover {
	background-color:#5d6d7e;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../icons/resources_ui.qrc">
     <normaloff>:/icons/actualizar.png</normaloff>:/icons/actualizar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDetalles_egresados">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  
	background-color: #2980b9;
    color: #FFFFFF;
    border: none;
    padding: 8px 8px;
    border-radius: 15px;
}
QPushButton:hover {
	background-color: #0D47A1
}</string>
   </property>
   <property name="text">
    <string>Detalles</string>
   </property>
   <property name="icon">
    <iconset resource="../icons/resources_ui.qrc">
     <normaloff>:/icons/buscar2_white.png</normaloff>:/icons/buscar2_white.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="lneBuscar_egresados">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>341</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>541</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #848f9d;
	color: #2d2d2d;
    border-radius: 12px;
    padding: 1px 9px;
    background-color: white;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Busqueda por cualquier dato</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblLogo_estu">
   <property name="geometry">
    <rect>
     <x>1039</x>
     <y>10</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>70</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../icons/resources_ui.qrc">:/logos/logo_escuela_sinFondo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frameFiltro_egresados">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>151</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
	background-color: white;
	border: 1.5px solid #2c3e50;
	border-radius: 12px;
}
QComboBox QAbstractItemView {
    border: 1px solid #ccc;
    border-radius: 5px;
    background-color: white;
    color: #333;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QComboBox" name="cbxFiltro_egresados">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>5</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border: transparent;</string>
    </property>
    <property name="iconSize">
     <size>
      <width>5</width>
      <height>5</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Todos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cédula</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nombres</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Apellidos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fecha nac.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Edad</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ciudad</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Género</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dirección</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ult. Grado</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ult. Seccion</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fecha Egreso</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>1029</x>
     <y>10</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>3</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>3</width>
     <height>61</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2d2d2d;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblTotalRegistros_egresados">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>590</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string>000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblTotalRegistros">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Total registros:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../icons/resources_ui.qrc"/>
 </resources>
 <connections/>
</ui>
